src/please.go

```diff
@@ -993,14 +974,100 @@ func execute(command string) bool {
 		}
 	}
 
-	success := buildFunctions[command]()
+	// TODO(bnm): This will eventually be replacing the original
+	var success bool
+	if command == "build" || command == "test" ||
+		command == "run" || command == "cover" ||
+		command == "rebuild" || command == "hash" ||
+		command == "parallel" {
+		initOpts, params := getInitOpsAndParams(command)
+		success = plz.Handle(command, *initOpts, params)
+	} else {
+		success = buildFunctions[command]()
+	}
```

Comment:
```
switch command {
case "build", "test", "run", "cover", "rebuild", "hash", "parallel":
  initOpts, params := getInitOpsAndParams(command) 
  success = plz.Handle(command, *initOpts, params)
default:
  success = buildFunctions[command]()
}
```

Reply:
This is temporary!:)

Reply:
Yeah, this PR is kind of like a proof of concept. I wanted to refactor please.go into its own package, so I can use those functionalities somewhere else later.
eventually `success = buildFunctions[command]()` will be replaced by
```
initOpts, params := getInitOpsAndParams(command)
success = plz.Handle(command, *initOpts, params)
```
So none of those if else statement will live long. :)